src/function.ts

```diff
@@ -80,7 +79,7 @@ export class DotNetFunction extends lambda.Function {
           : lambda.RuntimeFamily.OTHER;
         runtime = new lambda.Runtime(toolsRuntime, family);
       } else {
-        runtime = lambda.Runtime.DOTNET_6;
+        runtime = lambda.Runtime.DOTNET_8;
```

Comment:
Currently there is a test `integ.function-dotnet8` which tests explicitly set runtimes. With this change I would to suggest to change the test to `DOTNET_6` until we drop support in November.

Reply:
Yes, I agree with that. Will do that change as well.

Reply:
Now completed ✅